src/main/reactjs/public/i18n/koodisto/langs/en-GB.json

```diff
@@ -0,0 +1,205 @@
+{
+  "akt": {
+    "component": {
+      "publicTranslatorFilters": {
+        "languages": {
+          "BI": "Bislama",
+          "GD": "Gaelic, Scottish Gaelic",
+          "TE": "Telegu",
+          "KV": "Komi",
+          "HY": "Armenian",
+          "DZ": "Dzongkha; Bhutani; Butanese",
+          "FJ": "Fijian",
+          "MY": "Burmese, Myanmar",
+          "OC": "Occitan, Provencal",
```

Comment:
Ei varmaan mikään ykkösprioriteetin asia, mutta pohdin että tarvitaankohan UIn puolelle jokin mekanismi, jolla saadaan tarjottua näitä vaihtoehtoisia "nimiä" kielikoodeille muutenkin kuin yhtenä merkkijonona. En siis lähtisi tässä PR:ssä muuttamaan - varsinkin kun nämä kielet eivät ole (eurooppakeskeisestä näkökulmasta) kovinkaan valtakieliä - mutta ehkä tätä voi vastaisuudessa pohtia tarkemmin.